Lib/test/test_sqlite3/test_cli.py

```diff
@@ -200,5 +205,47 @@ def test_color(self):
             self.assertIn('\x1b[1;35mOperationalError (SQLITE_ERROR)\x1b[0m: '
                           '\x1b[35mnear "sel": syntax error\x1b[0m', err)
 
+@requires_subprocess()
+class CompletionTest(unittest.TestCase):
```

Comment:
```suggestion
@force_class_not_colorized
@requires_subprocess()
class CompletionTest(unittest.TestCase):
```
This will stop all color for all tests. (Like all other classes in the file.)

Reply:
Not work. This disables color in main process (the traceback becomes uncolored), but `run_pty()` still gives colored output containing control sequence. `run_pty()` runs in a subprocess.
![图片](https://github.com/user-attachments/assets/423f756a-0bb3-4fb5-b4dc-bcc02dfc0c14)

Reply:
Can we not then use `run_cli` (defined in the previous class) to make this much simpler overall?